src/os/bluestore/BlueStore.cc

```diff
@@ -6442,6 +6477,25 @@ int BlueStore::_mount(bool kv_only, bool open_db, bool do_bdev_expand)
     }
   }
 
+  {
```

Comment:
you can probably extend check_during_expand() function a bit (add 'error_if_expanding flag as a parameter?) and reuse it here instead of copy-pasting..